akka/shopping-cart-scala/build.sbt

```diff
@@ -2,9 +2,7 @@ name := "shopping-cart-service"
 
 organization := "com.lightbend.akka.samples"
 organizationHomepage := Some(url("https://akka.io"))
-licenses := Seq(
-  ("CC0", url("https://creativecommons.org/publicdomain/zero/1.0"))
-)
+licenses := Seq(("CC0", url("https://creativecommons.org/publicdomain/zero/1.0")))
```

Comment:
I am not really into this, but are we good regarding our change to BSL?

Reply:
Assume so, and that we're using CC0 across the Akka samples.